charts/karpenter/templates/clusterrole.yaml

```diff
@@ -40,3 +40,7 @@ rules:
     resources: ["mutatingwebhookconfigurations"]
     verbs: ["update"]
     resourceNames: ["defaulting.webhook.karpenter.k8s.aws"]
+  # Write
```

Comment:
Can you coordinate with @spring1843 to make sure this is landing in the right place?

Reply:
I think it's good here for now, this file is likely to stay here.